org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/JavaLanguageServerPlugin.java

```diff
@@ -113,6 +113,7 @@ public class JavaLanguageServerPlugin extends Plugin {
 	public static final String PLUGIN_ID = IConstants.PLUGIN_ID;
 
 	public static final String DEFAULT_MEMBER_SORT_ORDER = "T,SF,SI,SM,F,I,C,M"; //$NON-NLS-1$
+	public static final String DEFAULT_VISIBILITY_SORT_ORDER = "B,R,D,V"; //$NON-NLS-1$
```

Comment:
Although Eclipse's default is [`B,V,R,D`](https://github.com/eclipse-jdt/eclipse.jdt.ui/blob/50638e98c1c42ecd295f94fb4c2d4868944c4878/org.eclipse.jdt.ui/ui/org/eclipse/jdt/ui/PreferenceConstants.java#L4035), I agree it makes more sense to list in order of decreasing visibility by default.